var/spack/repos/builtin/packages/mpfr/package.py

```diff
@@ -39,8 +39,10 @@ class Mpfr(AutotoolsPackage):
     }
 
     for ver, checksum in patches.items():
-        patch('https://www.mpfr.org/mpfr-{0}/allpatches'.format(ver),
-              when='@' + ver, sha256=checksum)
+        patch(
```

Comment:
This looks the same but with a different style - if I'm right can you revert the changes to `mpfr`?